packages/nx/src/nx-cloud/generators/connect-to-nx-cloud/schema.json

```diff
@@ -21,6 +21,11 @@
       "description": "Hide formatting logs",
       "x-priority": "internal"
     },
+    "token": {
+      "type": "boolean",
+      "description": "Force local token creation",
+      "x-priority": "user"
```

Comment:
We can remove this. We set `important` to surface it higher in Nx Console, or `hidden` to hide it from Nx Console. It's not used for anything else.